### ISSUE TITLE
USER ISSUE: Null reference exception when chopping logs into smaller pieces after server restart

### ISSUE THREAD

**Issue by Ericinator** [StrangeLoopGames/EcoIssues#6969]
**Version:** 0.7.3.1 beta

**Steps to Reproduce:**
1) Felled about 7 trees without chopping any of the logs into smaller pieces. All trees were within close proximity of each other.
2) Server crashed and restarted.
3) Attempted to chop log into pieces.

**Expected behavior:**
Chop log into smaller pieces

**Actual behavior:**
The following error message popped up:
```
Caught exception

<size=60.00%>Object reference not set to an instance of an object.

   at TreeEntity.TrySliceTrunk(Player player, Single slicePoint)
   at Eco.Gameplay.Items.WeaponItem.OnActLeft(InteractionContext context)
   at Eco.Mods.TechTree.AxeItem.OnActLeft(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)
</size>
```

**Additional notes:**
- After the crash only 2 out of about 7 felled trees were affected by this problem. The other 5 behaved as expected.
- The latest 2 felled trees were affected.
- The game would crash on either chopping the large log (yellow outline) or when picking up a small log (green outline) that came from the same tree.
- After logging in after the crash, the action actually completes successfully, i.e. the log would be chopped or picked up. Meaning it was possible to eventually completely harvest the tree by repeatedly logging in, performing pickup/chop and then crashing.

**Comment by Ericinator** [comment]
Unfortunately I did not think of copying the exception for the pick up action at the time. I can't get it to happen again now.

**Comment by SeanAtr0n** [comment]
Thanks for the details, Fixed in 7.3.2
